governance/polls/Extend Strategic Happiness Core Unit (SH-001) Budget Until May Governance Cycle - April 25, 2022.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Extend Strategic Happiness Core Unit (SH-001) Budget Until May Governance Cycle - April 25, 2022
+summary: Signal your support or opposition to sending a one-time transfer of 35,000 DAI to the Strategic Happiness Core Unit
+discussion_link: https://forum.makerdao.com/t/signal-request-extend-shcu-sh-001-budget-until-may-governance-cycle/14511
+vote_type: Plurality Voting
+categories:
+   - Transfer
+   - Core Unit
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-25T16:00:00
+end_date: 2022-04-28T16:00:00
+---
+# Poll: Extend Strategic Happiness Core Unit (SH-001) Budget Until May Governance Cycle - April 25, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the MakerDAO Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, April 25 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following: 
+* Sending a one-time transfer of 35,000 DAI will be done to the Strategic Happiness Core Unit to cover the gap between budget cycles.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-extend-shcu-sh-001-budget-until-may-governance-cycle/14511) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* This change will be included in the weekly Executive Vote on the Friday following the conclusion of this poll.
```

Comment:
Can you change line 36 to "This change will be included in an upcoming Executive Vote. It is expected that this executive vote will take place within 30 days of this poll passing, absent external factors."

I need to update the template